### PR TITLE
Scala 2.11.7 -> Scala 2.11.8

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name := "akka-persistence-dynamodb"
 
-scalaVersion       := "2.11.7"
-crossScalaVersions := Seq("2.11.7", "2.12.0-M3")
+scalaVersion       := "2.11.8"
+crossScalaVersions := Seq("2.11.8", "2.12.0-M3")
```
Comment (patriknw) [inline build.sbt:4]: try to update this to 2.12.0-RC1 also, Akka 2.4.11 has a release for that

File: build.sbt [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 crossVersion       := CrossVersion.binary
 
 val akkaVersion = "2.4.2"
```
Comment (patriknw) [inline build.sbt:7]: please update to 2.4.11

**Comment by ktoso** [comment]
LGTM after Patrik's comment addressed :)

**Review by patriknw — APPROVED** [review]
LGTM, thanks @sullis

**Comment by patriknw** [comment]
Superseded by https://github.com/akka/akka-persistence-dynamodb/pull/53

**Pull request closed without merging**